webpack/scenes/Subscriptions/components/SubscriptionsTable/components/Table.js

```diff
@@ -27,7 +27,7 @@ const Table = ({
   const groupingController = {
     isCollapseable: ({ rowData }) =>
       // it is the first subscription in the group
-      rowData.id === groupedSubscriptions[rowData.product_id].subscriptions[0].id &&
+      rowData.id === 0 &&
```

Comment:
How is checking the id is 0 significant?

Reply:
Oh, I think it's so we don't get the '>' on the expanded rows also?

Reply:
@jturel yes that's exactly why.

Reply:
I wonder if there's some other distinguishing characteristic about the row (maybe something else we can add) to identify when we should collapse rather than id === 0. What if we had an Id later? That would break the functionality because it's not obvious that ID is significant in that way.

Reply:
What about putting a 'collapsible' field on the custom row we are building?

Reply:
@jturel Subscriptions already have an id field which starts from 1. So I just used id=0 to represent a generic row. But using a collapsible field sounds good to me.